.github/workflows/sonar-frontend.yml

```
@@ -0,0 +1,20 @@
+name: Sonar-Frontend
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, synchronize, reopened]
+jobs:
+  sonarcloud:
+    name: SonarCloud
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
+      - name: SonarCloud Scan
+        uses: SonarSource/sonarcloud-github-action@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information, if any
```

We don´t want comments in production I guess?
```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```